Rewrite/src/Rewrite.CSharp/Parser/CSharpParserVisitor.cs

```diff
@@ -397,7 +403,7 @@ public override J VisitIdentifierName(IdentifierNameSyntax node)
                 Core.Tree.RandomId(),
                 prefix,
                 Markers.EMPTY,
-                new JRightPadded<Expression>(fa.Target, Format(Leading(node.ArgumentList)), Markers.EMPTY),
+                new JRightPadded<Expression>(fa.Target, fa.Padding.Name.Before, Markers.EMPTY),
```

Comment:
Oops, this slipped in by accident. Let me know if I should revert.

Reply:
Doesn't matter, it's not correct anyways

Reply:
I saw you had changed it to something else in the mean time. That definitely didn't work for me. With this change I could at least get tests like this (which I believe you reported somewhere) working:
```csharp
void Test()
{
    ""blah"".Skip()
        .ToList();
}
```
But if you say it isn't correct, I believe you. I will let you take over.